be/src/storage/compaction_scheduler.cpp

```diff
@@ -224,27 +217,24 @@ std::shared_ptr<CompactionTask> CompactionScheduler::_try_get_next_compaction_ta
         int64_t tablet_id = compaction_candidate.tablet->tablet_id();
         CompactionType compaction_type = compaction_candidate.type;
         VLOG(2) << "try tablet:" << tablet_id << ", compaction type:" << to_string(compaction_type);
-        bool need_reschedule = true;
-        found = _can_do_compaction(compaction_candidate, &need_reschedule, &compaction_task);
-        if (need_reschedule) {
-            tmp_candidates.emplace_back(std::move(compaction_candidate));
-        } else {
-            DCHECK(found);
+
+        if (!_check_precondition(compaction_candidate)) {
+            LOG(INFO) << "check compaction precondition failed, candidate info: " << compaction_candidate.to_string();
```

Comment:
there may be many log here. consider to use VLOG?

Reply:
Updated.